src/components/Modal/Modal.stories.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import Modal from './Modal';
+import Button from '../Button/Button';
+
+export default {
+  title: 'Components/Modal/For Chromatic',
+  component: Modal,
+  parameters: {
+    chromatic: { delay: 1000 },
```

Comment:
@juanfabrega this doesn't seem to be working. I tried following these instructions:

https://www.chromatic.com/docs/delay#delay-all-stories-of-a-component